cypress/integration/home_page_spec.js

```diff
@@ -0,0 +1,50 @@
+import { aliasQuery, hasOperationName } from '../utils/graphql-test-utils';
+
+describe('Not Fur-gotten Home Page User Flow', () => {
+  beforeEach(() => {
```

Comment:
This beforeEach may not look like much, but it took about 6 human-hours to finally get this to work. We now know many, many ways NOT to intercept and stub GraphQL queries.